consensus/gxhash/algorithm_test.go

```diff
@@ -702,8 +702,8 @@ func TestConcurrentDiskCacheGeneration(t *testing.T) {
 		ParentHash:  common.HexToHash("0xd783efa4d392943503f28438ad5830b2d5964696ffc285f338585e9fe0a37a05"),
 		Coinbase:    common.HexToAddress("0xc0ea08a2d404d3172d2add29a45be56da40e2949"),
 		Root:        common.HexToHash("0x77d14e10470b5850332524f8cd6f69ad21f070ce92dca33ab2858300242ef2f1"),
-		TxHash:      common.HexToHash("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"),
-		ReceiptHash: common.HexToHash("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"),
+		TxHash:      common.EmptyTxsHash,
+		ReceiptHash: common.EmptyReceiptsHash,
```

Comment:
```suggestion
		TxHash:      types. EmptyTxRootOriginal,
		ReceiptHash: types. EmptyTxRootOriginal,
```
Well this code is dead anyway..